var/spack/repos/builtin/packages/flux-core/package.py

```diff
@@ -45,6 +47,7 @@ class FluxCore(AutotoolsPackage):
     variant('docs', default=False, description='Build flux manpages')
     variant('cuda', default=False, description='Build dependencies with support for CUDA')
 
+    depends_on("ncurses@6.2", when="@0.32.0:,master")
```

Comment:
```suggestion
    depends_on("ncurses@6.2", when="@0.32.0:")
```

`Version('master').satisfies('@0.32.0:')`, there's no need to list it separately.

The "special" versions are `develop > main > master > head > trunk > 9999999999`.